### PR TITLE
DOC-7557 Added new infrastructure alert conditions procedure

### PR DESCRIPTION
**Pull request by paperclypse**
I added a new procedure to describe how to create an alert condition in the Infrastructure settings UI.

### CODE REVIEW

**Comment by paperclypse** [comment]
This one is still a work-in-progress. Not ready for PR review.

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx
Comment (barbnewrelic) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:82]: I wonder if you should call this id "integration" rather than "infrastructure-settings" because the original "integration" ID (line 57) has been deleted?

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx
Comment (barbnewrelic) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:58]: I don't think Cloud Integration needs to be initial caps.
Maybe reword "then it won't be selectable for alerting in the UI" to "then you can't select it in the UI" or "then it won't be an alerting option in the UI"?
Last sentence: I think I know what "silent attributes" means, but it caught me by surprise. Maybe reword to something like:
If New Relic hasn't received a cloud integration service's attribute in the past 60 minutes, we refer to this as a "silent attribute," and it won't be available to use as an alert condition in the UI. In this situation, you can use the API to create alert conditions for silent attributes.

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx
Comment (barbnewrelic) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:78]: Maybe consider breaking this into individual steps rather than a single long paragraph for easier skimming?

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx
Comment (barbnewrelic) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:71]: I don't have a test account to look at policy setup in UI but am assuming "product" is still the right word/ field name?

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx
Comment (barbnewrelic) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:83]: Maybe change title to "Use infrastructure settings for integrations" so the collapser is clear what problem this solves?

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx
Comment (barbnewrelic) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:94]: thresholds, and then
^^ add comma like step 2 does

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx
Comment (barbnewrelic) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:96]: Seems like 2 separate thoughts. "optional. You" ?

**Review by barbnewrelic — CHANGES_REQUESTED** [review]
I like the flow of this, and it looks fine in my localhost. I've made a lot of minor copyedit suggestions, take 'em or leave 'em! Then feel free to merge your PR with your admin privileges. No need for second review from me. Nice job!

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx
Comment (paperclypse) [reply src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:58]: Good call!

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx
Comment (paperclypse) [reply src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:71]: Yup, it's still "product" in the UI

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx
Comment (paperclypse) [reply src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:78]: I was considering this, but your opinion has tipped me over to doing it :)

**Comment by paperclypse** [comment]
Good suggestions! Thanks, @barbnewrelic!

**Pull request merged**
